projects/jobs/jobs.groovy

```diff
@@ -23,13 +23,52 @@ readFileFromWorkspace("${WORKSPACE}/cartridges.txt").eachLine { line ->
 
 // Jobs
 def loadCartridgeJob = freeStyleJob(cartridgeManagementFolderName + "/Load_Cartridge")
-def loadCartridgeCollectionJob = workflowJob(cartridgeManagementFolderName + "/Load_Cartridge_Collection")
-
-
+def loadCartridgeCollectionJob = workflowJob(cartridgeManagementFolderName + "/Load_Cartridge_Collection")/var/jenkins_home/userContent
```

Comment:
Does this mean Load_Cartridge has to be run on master as it depends on the file system?